arm-documentdb/2015-04-08/examples/DocumentDBDatabaseAccountCreateMax.json

```diff
@@ -0,0 +1,82 @@
+{
+  "parameters": {
+    "accountName": "ddb1",
+    "resourceGroupName": "rg1",
+    "api-version": "2015-04-08",
+    "subscriptionId": "subid",
+    "location": "westus",
```

Comment:
general comment: please add the body parameter under the name of the body parameter defined in swagger.
```json5 
"createUpdateParameters": { . . .  entire body parameter here...}
```